mobile/native/src/api.rs

```diff
@@ -80,7 +80,7 @@ pub struct WalletInfo {
 #[derive(Clone, Debug, Default)]
 pub struct Balances {
     pub on_chain: u64,
-    pub lightning: u64,
+    pub off_chain: u64,
```

Comment:
nit: it's weird to me that on the coordinator you've changed the model from offchain to lightning and here you do it the other way around. It would be nice to have a common term for this kind of balance.

Reply:
I was thinking that we will release a new app to the user and force-close all channels to the users - hence, they don't need this anymore. For the coordinator in contrast we might want to see if there are still some channels open.